tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -0,0 +1,1437 @@
+  }
+
+  /**
+   * Disables rule.
+   *
+   * @param name either a fully qualified name or a simple entity name
+   * @param expectedExitCode (optional) the expected exit code for the command
+   * if the code is anything but DONTCARE_EXIT, assert the code is as expected
+   */
+  override def disable(name: String, expectedExitCode: Int = SUCCESS_EXIT)(implicit wp: WskProps): RestResult = {
+    changeRuleState(name, "inactive")
+  }
+
+  /**
+   * Checks state of rule.
+   *
+   * @param name either a fully qualified name or a simple entity name
+   * @param expectedExitCode (optional) the expected exit code for the command
+   * if the code is anything but DONTCARE_EXIT, assert the code is as expected
+   */
+  override def state(name: String, expectedExitCode: Int = OK.intValue)(implicit wp: WskProps): RestResult = {
+    get(name, expectedExitCode = expectedExitCode)
+  }
+
+  def changeRuleState(ruleName: String, state: String = "active")(implicit wp: WskProps): RestResult = {
+    val enName = entityName(ruleName)
+    val path = getNamePath(noun, enName)
+    val bodyContent = JsObject("status" -> state.toJson)
+    val resp = requestEntity(POST, path, body = bodyContent.toString)
+    new RestResult(resp.status, getRespData(resp))
+  }
+}
+
+class WskRestActivation extends RunWskRestCmd with HasActivationRest with WaitFor with BaseActivation {
+
+  protected val noun = "activations"
+
+  /**
+   * Activation polling console.
+   *
+   * @param duration exits console after duration
+   * @param since (optional) time travels back to activation since given duration
+   */
+  override def console(duration: Duration, since: Option[Duration] = None, expectedExitCode: Int = SUCCESS_EXIT)(
+    implicit wp: WskProps): RestResult = {
+    var sinceTime = System.currentTimeMillis()
+    val utc = Instant.now(Clock.systemUTC()).toEpochMilli
+    sinceTime = since map { s =>
+      sinceTime - s.toMillis
+    } getOrElse sinceTime
+    val pollTimeout = duration.toSeconds
+    waitForActivationConsole(duration, Instant.ofEpochMilli(sinceTime))
+  }
+
+  /**
+   * Lists activations.
+   *
+   * @param filter (optional) if define, must be a simple entity name
+   * @param limit (optional) the maximum number of activation to return
+   * @param since (optional) only the activations since this timestamp are included
+   * @param expectedExitCode (optional) the expected exit code for the command
+   * if the code is anything but DONTCARE_EXIT, assert the code is as expected
+   */
+  def listActivation(filter: Option[String] = None,
+                     limit: Option[Int] = None,
+                     since: Option[Instant] = None,
+                     docs: Boolean = true,
+                     expectedExitCode: Int = SUCCESS_EXIT)(implicit wp: WskProps): RestResult = {
+    val entityPath = Path(s"${basePath}/namespaces/${wp.namespace}/$noun")
+    var paramMap = Map("skip" -> "0".toString, "docs" -> docs.toString) ++ {
```

Comment:
`"0"` is already a string. Shouldn't need to use `toString` there.

Reply:
Done.